tiledb/ml/models/model_list.py

```diff
@@ -0,0 +1,53 @@
+import tiledb.cloud
+
+FILETYPE_ML_NODEL = "ml_model"
+
+
+class ModelList:
```

Comment:
Not sure about this class in general.. 3 methods that take no arguments and only `models` seems to be the public one(?). I think it could be a standalone function instead.

Reply:
It is one function now, files is removed in favor of `cloud_utils`